### ISSUE TITLE
`Auto upload`: `Remove from camera roll` _older than n_.

### ISSUE THREAD

**Issue by jtagcat** [nextcloud/ios#1599]
### Expected behaviour

Option to define a period of hours/days/months, where photos are not removed from camera roll.  
As an example, setting it to 30d, all photos-videos older than and equal to 30d will be cloud-only. Photos-videos newer than 30d will still be uploaded, but not yet removed from the device.

### Actual behaviour

All photos are deleted locally. Sharing is quite difficult, if the image is not local to the device, or on iCloud Photos.

**Comment by eharris** [comment]
I agree that this might be a nice feature to have.  But there are several existing bugs relating to loss of photos from the auto-upload process (e.g. #1266, #1550, and others), and there could always be future as-yet undiscovered bugs) so this shouldn't be implemented unless there is also code to actively check that each file exists and is complete and identical in your Nextcloud storage before it is ever considered for deletion on the local device.

**Comment by pipaliyachirag** [comment]
do we have this feature now? i think we have pretty stable base to implement this feature
